docker/Dockerfile.gateway

```diff
@@ -25,7 +25,7 @@ FROM base AS build
 
 WORKDIR /workspace
 
-RUN apt-get install -y libfuse3-dev && \
+RUN apt-get install -y libfuse3-dev libbtrfs-dev libgpgme-dev && \
```

Comment:
I think there may have been an issue with libgpg + cedana, we should look into if it causes any issues with C/R